test/onnx/expect/TestOperators.test_dyn_arange.expect

```diff
@@ -48,8 +48,8 @@ graph {
       name: "value"
       t {
         dims: 1
-        data_type: 7
-        raw_data: "\001\000\000\000\000\000\000\000"
+        data_type: 1
```

Comment:
What is going on here? Why does the onnx expect change? Shouldn't onnx follow the python frontend?

Reply:
i dont know, it's possible it has something to do with the dtype changing

Reply:
I think it should be the same, could you run `python test/onnx/test_operators.py --accept` to update the expect files?

From the CI, seems they should stay the same.

Reply:
I updated this because it was failing previously